source/documentation/runbooks/002-populate-an-environment.html.md.erb

```diff
@@ -0,0 +1,55 @@
+---
+title: Populate an environment
+last_reviewed_on: 2024-03-27
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Setup metadata ingestion
+
+Checkout [data-catalogue-metadata](https://github.com/ministryofjustice/data-catalogue-metadata) (internal repo) and follow the instructions in that repo:
+
+- Setup the Dathaub CLI
+- Run ingestions handled by the python script
+- Run one-off command line ingestions
+- Setup scheduled ingestion recipes in Datahub
+
+### Scheduled ingestion checklist
+
+You should have the following ingestions scheduled to run from Datahub itself:
+
+- DBT
+
+### Entity checklist
+
+After populating an environment you should expect to see the following entities in Datahub:
+
+- Glossary groups and terms
+- Domains
+- Users
+- Datasets (tables from Analytical platforms and create-a-derived-table)
+- Containers (databases from Analytical platform and create-a-derived-table)
+- Data Products
```

Comment:
If we are moving away from data products, perhaps this could be databases?

Reply:
We still have a few examples, so I'll leave this until we remove them

Reply:
Databases are within 'containers'